src/reports.cpp

```diff
@@ -786,7 +786,8 @@ static int attack_info(const reports::context& rc, const attack_type &at, config
 		int specials_damage = at.modified_damage();
 		int damage_multiplier = 100;
 		const_attack_ptr weapon  = at.shared_from_this();
-		int tod_bonus = combat_modifier(get_visible_time_of_day_at(rc, hex), u.alignment(), u.is_fearless());
+		unit_alignments::type alignment = weapon->alignment_in_attack();
```

Comment:
Here it would be useful to use weapon_- or attack_alignment as the variable name.

Reply:
done